backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -112,11 +112,29 @@ export class TransactionResolver {
     const self = new User(user)
     const transactions: Transaction[] = []
 
+    const transactionLinkRepository = getCustomRepository(TransactionLinkRepository)
+    const { sumHoldAvailableAmount, sumAmount, lastDate, firstDate } =
+      await transactionLinkRepository.summary(user.id, now)
```

Comment:
move this in line 123-124 into the `if (!onlyCreations && currentPage === 1 && order === Order.DESC)` if possible